tour-of-heroes/protractor-saucelabs.conf.js

```diff
@@ -0,0 +1,48 @@
+var defaultConfig = require('./protractor.conf.js').config;
+
+var testName = 'Tour of Heroes Smoke Tests'
+
+var ciConfig = {
+  // Note that seleniumAddress must be unset in order for SauceLabs to work.
+  seleniumAddress: false,
+  sauceUser: process.env.SAUCE_USERNAME,
+  sauceKey: process.env.SAUCE_ACCESS_KEY,
+
+  directConnect: false,
```

Comment:
Instead of trying to override an existing with an Object.assign, could we instead do something like:

```
var defaultConfig = require('./protractor.conf.js').config;

defaultConfig.directConnect = undefined;
defaultConfig.seleniumAddress = undefined;
defaultConfig.sauceUser
...

exports.config = defaultConfig;
```

similar to https://github.com/angular/protractor/blob/6a667c8a3211ac451a264544bb6f44dfe29f7b74/spec/ciNg2Conf.js